config/mesh_defaults.sh

```diff
@@ -742,6 +742,13 @@ case $GRIDNAME in
       UNITOFFSETFILE=null
       ;;
    *)
-      warn "cycle $CYCLE: $SCENARIO: $THIS: Mesh GRIDNAME $GRIDNAME was not recognized."
+      localMeshParameters=$SCRIPTDIR/local/config/mesh_defaults.sh
```

Comment:
Was this going to include a configurable base directory envar? We'll want to make sure we add this to be tracked by the profiles; the list of envars the shell will save is tracked in `_ASGS_EXPORTED_VARS_` so any "local" base dir variable name should be added there, see https://github.com/StormSurgeLive/asgs/blob/master/cloud/general/asgs-brew.pl#L452

This way, setting a "local" director per profile is then possible.

Reply:
I'm not confident that I can do this without further guidance ... let's discuss.